### PR TITLE
Merge open source checklist compliance changes with develop branch

### PR DESCRIPTION
**Pull request by prempratapsingh**
## Description
Added required settings and configuration for TrustedShop's open source project checklist - https://wiki.trustedshops.com/display/TrustProductsServices/Open+Source+Guild+-+Project+Check+List 

1. Added header template for inserting license text in new files created
2. Update existing files with license header text
3. Add contributions.md file
4. Add renovate.json file 
5. Removed references of client id and secret from the compliance project and updated readme with details about how to  contact mobile team to obtain demo client and secret for testing
6. Add template for bugs and feature requests

### CODE REVIEW

File: Example/Trustylib-Integration-UIKit/Trustylib-Integration-UIKit.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -5,7 +5,7 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/trustedshops-public/etrusted-ios-trustbadge-library",
       "state" : {
-        "revision" : "3a2861984adb81a1aeac8a8d2c82c18db40ea7b3",
+        "revision" : "eafb1f755bd2097a9fee39161efcb41e92dc8eff",
```
Comment (superus8r) [inline Example/Trustylib-Integration-UIKit/Trustylib-Integration-UIKit.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:8]: question: what is this parameter used for?

File: Sources/Trustylib/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,10 +1,28 @@
...
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+//
+//  Created by Prem Pratap Singh on 07/11/22.
+//
 
 "Excellant" = "Excellant";
```
Comment (superus8r) [inline Sources/Trustylib/Localization/en.lproj/Localizable.strings:27]: chore: it seems like there is a typo in this word, if that's the case, please consider a correction 🙂

**Review by superus8r — CHANGES_REQUESTED** [review]
There seem to be a tiny typo, everything else looks great! 🎉

File: Example/Trustylib-Integration-UIKit/Trustylib-Integration-UIKit.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved (hunk 1 above)
Comment (prempratapsingh) [reply Example/Trustylib-Integration-UIKit/Trustylib-Integration-UIKit.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:8]: This must be Swift package manager related parameter which is auto updated by XCode.

File: Sources/Trustylib/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (prempratapsingh) [reply Sources/Trustylib/Localization/en.lproj/Localizable.strings]: Yes, there is a type. Nice catch Ali 👍🏽 
I have fixed it, please review.

**Pull request merged**